src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,91 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task{
+
+    private static final DateTimeFormatter DATE_FORMAT_INP = DateTimeFormatter.ofPattern("yyyy-MM-dd' 'HH:mm");
+    private static final DateTimeFormatter DATE_FORMAT_OUT = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
+    private LocalDateTime deadline;
+
+    private static final String SYMBOL = "D";
+
+    /**
+     * Constructor for Deadline.
+     * This constructor will be used when given
+     * description and date as the parameter.
+     *
+     * @param des description for the task.
+     * @param dl deadline date.
+     */
+    public Deadline(String des, LocalDateTime dl) {
+        super(des);
+        deadline = dl;
+    }
+
+    /**
+     * Constructor for Deadline.
+     * This constructor will be used when given
+     * mark status, description, and date as the parameter.
+     *
+     * @param st mark status
+     * @param des description for the task.
+     * @param dl deadline date.
+     */
```

Comment:
Is this how the method should be indented? I believe you are missing 4 spaces.

Reply:
Will review it later. Thanks